src/Sulu/Component/Util/XmlUtil.php

```diff
@@ -0,0 +1,43 @@
+<?php
+
+namespace Sulu\Component\Util;
+
+class XmlUtil
+{
+    /**
+     * Returns value of path.
+     *
+     * @param string $path
+     * @param \DOMXPath $xpath
+     * @param \DomNode $context
+     * @param mixed $default
+     *
+     * @return bool|null|string|mixed
+     */
+    public static function getValueFromXPath($path, \DOMXPath $xpath, \DomNode $context = null, $default = null)
+    {
+        try {
+            $result = $xpath->query($path, $context);
+            if ($result->length === 0) {
+                return $default;
+            }
+
+            $item = $result->item(0);
+            if ($item === null) {
+                return $default;
+            }
+
+            if ('true' === $item->nodeValue) {
```

Comment:
This wouldn't work when somebody actually wanted the word `true`.

It would be better to declare the type of the thing you want, e.g. `->getValue('boolean', $foobar ...);`

Would consider adding that feature to `phpbench/dom` as referenced above to have something like `-`-`->evaluate($expression, $type, $default)`